services/proxy/README.md

```diff
@@ -13,6 +13,35 @@ The following request authentication schemes are implemented:
 -   Signed URL
 -   Public Share Token
 
+## Configuring routes
+
+The proxy handles routing to all endpoints that ocis offers. The current default routes can be found here https://github.com/owncloud/ocis/blob/master/services/proxy/pkg/config/defaults/defaultconfig.go
+Due to complexity of the configuration, new routes can only be defined in the yaml file.
+For overwriting the default routes use:
+```yaml
+policies:
+  - name: ocis
+    routes:
+      - endpoint: /
+        service: com.owncloud.web.web
+      - endpoint: /dav/
+        service: com.owncloud.web.ocdav
+```
+For adding additional routes to the default routes use:
+```yaml
+additional_policies:
+  - name: ocis
+    routes:
+      - endpoint: /custom/endpoint
+        service: com.owncloud.cutom.custom
+```
+A route has the following configurable parameters:
+```yaml
+endpoint: "" # the url that should be routed
+service: "" # the service the url should be routed to
+unprotected: false # skip authorization
```

Comment:
```suggestion
endpoint: ""       # the url that should be routed
service: ""        # the service the url should be routed to
unprotected: false # skip authorization
```
Renders hashes underneath making the keys better readble
What happens with `unprotected` true?

Reply:
`unprotected: true` => No authorization checks. This is an open endpoint
`unprotected: false` => Authorization is checked on request, user needs an auth header to access this endpoint